scripts/qa_crowbarsetup.sh

```diff
@@ -1281,6 +1281,11 @@ function onadmin_set_source_variables()
             CLOUDSLE12TESTISO="CLOUD-7-TESTING-$arch*Media1.iso"
             CLOUDLOCALREPOS="SUSE-OpenStack-Cloud-7-devel"
         ;;
+        mitakacloud7)
+            CLOUDSLE12DISTPATH=/ibs/Devel:/Cloud:/7:/Mitaka/images/iso
+            CLOUDSLE12DISTISO="SUSE-OPENSTACK-CLOUD-7-$arch*Media1.iso"
+            CLOUDLOCALREPOS="SUSE-OpenStack-Cloud-7-official"
+        ;;
```

Comment:
Thats right, we treat it more like cloudsource =~ susecloud7 (e.g. we skip cct). I think thats okay..